modules/test-utils/src/main/java/org/jboss/wsf/test/AppclientHelper.java

```diff
@@ -193,6 +193,7 @@ private static AppclientProcess newAppclientProcess(final String archive, final
             javaOptsValue.append(acDeubOpts).append(" ");
          }
          pb.environment().put("JAVA_OPTS", javaOptsValue.toString());
+         pb.environment().put("MODULE_OPTS", "--add-modules=java.se");
```

Comment:
yes, after having applied the changes that Rosta is suggesting

Reply:
what about reusing "modular.jdk.args" property set by Maven based on java version instead of the own version detect logic , eg.
```java
javaOptsValue.append(" ").append(System.getProperty("modular.jdk.args", ""));
pb.environment().put("JAVA_OPTS", javaOptsValue.toString());
```
btw once the appclient script is fixed, this should be removed